packages/libs/core/src/handle/unauthorized.ts

```diff
@@ -0,0 +1,9 @@
+import { setHeadersFromRoute } from "./headers";
+import { Event, UnauthorizedRoute } from "../types";
+
+export const unauthorized = (event: Event, route: UnauthorizedRoute) => {
+  setHeadersFromRoute(event, route);
+  event.res.statusCode = route.status;
+  event.res.statusMessage = route.statusDescription;
+  event.res.end("Unauthorized");
```

Comment:
Maybe unauthorized can be empty (I think I might have put this before)? Then the browser can render a nicer page.

Reply:
Sure, I can change that. (Although the feature itself is supposed to be dev only.)